extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -381,17 +322,15 @@ public long getDislikeCount() throws ParsingException {
     @Override
     public String getUploaderUrl() throws ParsingException {
         assertPageFetched();
-        String uploaderId = null;
         try {
-            uploaderId = getVideoSecondaryInfoRenderer().getObject("owner").getObject("videoOwnerRenderer")
-                    .getObject("navigationEndpoint").getObject("browseEndpoint").getString("browseId");
+            String uploaderUrl = getUrlFromNavigationEndpoint(getVideoSecondaryInfoRenderer()
+                    .getObject("owner").getObject("videoOwnerRenderer").getObject("navigationEndpoint"));
+            if (uploaderUrl != null) return uploaderUrl;
+        } catch (Exception ignored) {}
+        try {
+            String uploaderId = playerResponse.getObject("videoDetails").getString("channelId");
+            if (uploaderId != null) return "https://www.youtube.com/channel/" + uploaderId;
         } catch (Exception ignored) {}
-        if (uploaderId == null) {
-            try {
-                uploaderId = playerResponse.getObject("videoDetails").getString("channelId");
-            } catch (Exception ignored) {}
-        }
-        if (uploaderId != null) return "https://www.youtube.com/channel/" + uploaderId;
         throw new ParsingException("Could not get uploader url");
```

Comment:
Same as above to save `if` checks and improve code readability, I'd move `if (uploaderId == null) throw new ParsingException("Could not get uploader url");` at the end of the `if` just above